dataactcore/migrations/versions/c75d57250419_.py

```diff
@@ -0,0 +1,49 @@
+"""empty message
```

Comment:
There should always be a message in migrations so we know what's happening. The name of the migration file should be a portion of this message. Here it can be "Adding state name columns to all fsrs tables" or something